examples/backend/src/main.ts

```diff
@@ -0,0 +1,28 @@
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from './app/app.module';
+import { Environment, EnvironmentType } from './config/environment';
+import { swaggerSetup } from './config/swagger';
+
+async function bootstrap() {
+    const app = await NestFactory.create(AppModule);
+
+    // Handle Errors
```

Comment:
Why this comment code?

Reply:
The backend comes from another project, this part was pending to be moved to the nest package. I've created a factory method on the same class to ease the instantiation. Changed in 08869969fe2f47170f45ce4b439ecf83036372ab.